README.md

```diff
@@ -24,7 +24,7 @@ You must have [Docker](https://www.docker.com/get-started/) installed. This Even
 The first step is to **create a configuration file** for the emulator to know the topics, and for each topic, the webhooks to call when an event is published.
 Create a configuration file named `appsettings.json` somewhere on your computer, for instance: `C:\eventgridemulator\appsettings.json`.
 
-It should look like this for Push Model:
+### Push Delivery:
```

Comment:
Maybe split in 4 section, that would make it clearer the disctinction and relation betwenn all

- Publish event on Custom Topics
- Publish event on Namespace Topics
- Pull Model
- Push Model

Reply:
Namespace topics only supports the CloudEvents schema
![image](https://github.com/gsoft-inc/wl-eventgrid-emulator/assets/158102624/deb0ed10-9087-4536-8d91-715c40ee225b)